src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountExtension.groovy

```diff
@@ -33,8 +33,9 @@ class DexMethodCountExtension {
     private int dxTimeoutSec = 60;
     private String teamCitySlug = null
     private boolean runOnEachAssemble = true
+    int maxMethodCount
```

Comment:
Let's rewrite this in the future to be more "groovy-like". We can cut back on alot of the Java code and remove modifiers by writing just "pure" groovy.